library/objective-c/EnvoyConfiguration.m

```diff
@@ -9,11 +9,12 @@ - (instancetype)initWithStatsDomain:(NSString *)statsDomain
                   dnsRefreshSeconds:(UInt32)dnsRefreshSeconds
        dnsFailureRefreshSecondsBase:(UInt32)dnsFailureRefreshSecondsBase
         dnsFailureRefreshSecondsMax:(UInt32)dnsFailureRefreshSecondsMax
-                        filterChain:(NSArray<EnvoyHTTPFilterFactory *> *)httpFilterFactories
+                platformFilterChain:(NSArray<EnvoyHTTPFilterFactory *> *)httpFilterFactories
                   statsFlushSeconds:(UInt32)statsFlushSeconds
                          appVersion:(NSString *)appVersion
                               appId:(NSString *)appId
-                    virtualClusters:(NSString *)virtualClusters {
+                    virtualClusters:(NSString *)virtualClusters
+                  nativeFilterChain:(NSArray<EnvoyNativeFilterConfig *> *)nativeFilterChain {
```

Comment:
nit: I think it'd be nice to list native/platform filters together in the parameter order